students/LouReis/Lesson03/slicing.py

```diff
@@ -0,0 +1,64 @@
+# slicing.py
+#
+#
+# Coder: LouReis
+
+def copy_sequence(seq):
+    """Return a copy of the sequence entered"""
+    return seq
+
+# Below is test code to convert input to a string for easier manipulation
+#def copy_first_last(seq):
+#    """Return the sequence with first and last items entered swapped"""
+#    seq = str(seq)
+#    seq = seq[-2]+seq[2:-2]+seq[1]
+#    return seq.join()
+
+def copy_first_last(seq):
+    """Return the sequence with first and last items entered swapped"""
+    if type(seq) is tuple:
+        return (seq[-1],)+seq[1:-1]+(seq[0],)
+    elif type(seq) is str:
+        return (seq[-1]+seq[1:-1]+seq[0])
```

Comment:
use slicing instead of indexing; if you only use slicing then you don't need to check the data type